src/full_node/full_node_api.py

```diff
@@ -81,12 +80,16 @@ async def request_peers(self, _request: full_node_protocol.RequestPeers, peer: w
     @peer_required
     @api_request
     async def respond_peers(
-        self, request: introducer_protocol.RespondPeers, peer: ws.WSChiaConnection
+        self, request: full_node_protocol.RespondPeers, peer: ws.WSChiaConnection
     ) -> Optional[Message]:
         if self.full_node.full_node_peers is not None:
-            await self.full_node.full_node_peers.respond_peers(request, peer.get_peer_info(), False)
-        if peer.connection_type is NodeType.INTRODUCER:
-            await peer.close()
+            if peer.connection_type is NodeType.INTRODUCER:
```

Comment:
we still want to close introducer peer connection even if self.full_node.full_node_peers is None